test/test_cli.py

```diff
@@ -63,11 +67,20 @@
 assert_equal(value, "40 60", "Checking complex input failed!")
 
 assert_salty_deploy(
```

Comment:
I think it should be commented here that the purpose of the salty test isn't just testing salty functionality, but also testing events functionality.